meta-mender-commercial/recipes-mender/mender-monitor/mender-monitor.inc

```diff
@@ -18,9 +24,20 @@ do_version_check() {
 }
 addtask do_version_check after do_unpack before do_install
 
+SYSTEMD_SERVICE_${PN}_mender-systemd = "mender-monitor.service"
+
 do_install() {
     oe_runmake \
         -C ${S} \
         DESTDIR=${D} \
         install
```

Comment:
Does this `install` target also include `install-systemd`? Then we cannot use it and we need to specify all the other `install-*` targets here.

Reply:
I would prefer removing install-systemd from the Makefile from monitor-client -- it makes better sense, can it be like that?

Reply:
As you wish.

Reply:
aligned with https://github.com/mendersoftware/monitor-client/pull/17